### PR TITLE
🌱  Add sbueringer to cluster-api-reviewers

### PR DESCRIPTION
**Pull request by CecileRobertMichon**
**What this PR does / why we need it**: @sbueringer has made quality [reviews](https://github.com/kubernetes-sigs/cluster-api/pulls?q=is%3Apr+assignee%3Asbueringer+is%3Aclosed), [contributions](https://github.com/kubernetes-sigs/cluster-api/pulls?q=is%3Apr+is%3Aclosed+author%3Asbueringer), and [investigations](https://github.com/kubernetes-sigs/cluster-api/issues?q=is%3Aissue+author%3Asbueringer+) to CAPI for past few months. This PR adds him to the reviewers for CAPI. 

Thank you all your contributions so far @sbueringer!

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

### CODE REVIEW

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -27,9 +27,8 @@ aliases:
 
   # folks who can review and LGTM any PRs in the repo
   cluster-api-reviewers:
-  - CecileRobertMichon
-  - vincepri
```
Comment (CecileRobertMichon) [inline OWNERS_ALIASES:31]: @vincepri removed you and I from reviewers since we're both maintainers, I don't think there is value in having us listed twice

**Comment by elmiko** [comment]
+1

**Comment by fabriziopandini** [comment]
Thanks @sbueringer for the great contribution so far!
/lgtm

**Comment by detiber** [comment]
+1, thanks for all your hard work @sbueringer

**Comment by enxebre** [comment]
+1

**Comment by randomvariable** [comment]
+1

File: OWNERS_ALIASES (hunk 1 above)
Comment (vincepri) [reply OWNERS_ALIASES:31]: IIRC reviewers get randomly assigned to PR reviews, but that list usually doesn't include approvers/maintainers as well.

Comment (sbueringer) [reply OWNERS_ALIASES:31]: I think you're right. The default logic (which should be active in our repo, I assume we're using blunderbuss) is:
* pick two reviewers
* if there aren't enough reviewers pick approvers

So I think with the config on this PR always Joel and I will be assigned as reviewers. But I might be wrong

Some links:
https://github.com/kubernetes/test-infra/blob/1a550c281187f9eb11bdbcb4272669f213f048dd/config/prow/plugins.yaml#L202-L204
https://github.com/kubernetes/test-infra/blob/c273d05a692dd96bd15344528aa90cb9978d8bb5/prow/plugins/config.go#L143-L147
https://github.com/kubernetes/test-infra/blob/895e1d55e1593b9f550aff7c1c25ae6dcd949b96/prow/plugins/blunderbuss/blunderbuss.go#L233-L239

Comment (CecileRobertMichon) [reply OWNERS_ALIASES:31]: Does that really matter to you @vincepri ? I don't know about you but I'm going to go through the PR queue and review PRs regardless of whether the bot assigned me or not... If do we care about assigning approvers as reviewers then it should be the full approver list, not just us 2 IMO

Comment (sbueringer) [reply OWNERS_ALIASES:31]: That's a good point. I personally also work based on Github notifications and a watch on the whole repo and not based on Prow assignments.

Comment (randomvariable) [reply OWNERS_ALIASES:31]: I've always wondered why @vincepri gets assigned reviews on CAPA PRs even though I took him out. Where would I put `ExcludeApprovers`  in the config?

Comment (sbueringer) [reply OWNERS_ALIASES:31]: As far as I know the blunderbuss plugin is configured globally in prow: (in contrast to approve / blockade)
* https://github.com/kubernetes/test-infra/blob/c273d05a692dd96bd15344528aa90cb9978d8bb5/prow/plugins/config.go#L65-L67
* https://github.com/kubernetes/test-infra/blob/1a550c281187f9eb11bdbcb4272669f213f048dd/config/prow/plugins.yaml#L202-L204
So I think this needs a change in Prow, to be able to also configure this on an org or repo level.

**Comment by CecileRobertMichon** [comment]
rebased

File: OWNERS_ALIASES (hunk 1 above)
Comment (vincepri) [reply OWNERS_ALIASES]: > Does that really matter to you @vincepri ?

Nope, mostly wanted to make sure we were all aware

**Comment by CecileRobertMichon** [comment]
> /hold for lazy consensus until Friday

/hold cancel

**Comment by fabriziopandini** [comment]
/approve
/lgtm
Lazy consensus expired and I don't see objections
Congrats @sbueringer !

**Pull request merged**